Source/Core/Core/PowerPC/XER.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+namespace PowerPC
+{
+// XER, reformatted into byte fields for easier access.
+struct XER
+{
+  u8 ca;
+  u8 so_ov;  // format: (SO << 1) | OV
```

Comment:
Could we use `BitField` for this, or does that break standard layout?

Reply:
Oops, I only just noticed you commented; I tried and it doesn't trigger the static_assert. Now if only I knew a game that uses SO/OV (and not just the fourth bit in each CR field) so I can test it :S